decidim-admin/spec/features/admin_manages_participatory_process_attachments_spec.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require_relative "../shared/participatory_admin_shared_context"
+require_relative "../shared/manage_process_attachments_examples"
+
+describe "Admin manages participatory process attachments", type: :feature do
+  include_context "participatory process admin"
+  it_behaves_like "manage process attachments examples"
```

Comment:
I'm not sure I like duplicating this just to double-check roles are working as expected. Looks like we're missing a unit test somewhere.